controllers/users/index.js

```diff
@@ -0,0 +1,29 @@
+const express = require('express');
+const Users = require('../../models/users');
+
+const router = express.Router();
+
+router.get('/:id', (req, res) => {
+    Users.getById(req.params.id).then((user) => {
+        res.json(user);
+    });
+});
+
+router.get('/getByUsername', (req, res) => {
```

Comment:
I'd recommend changing these slightly as it doesn't follow REST best practices. You would want the route to be `/` but then have a query parameter instead that switches the query type.

`/?username=joe.bloggs`
`/?email=joe.bloggs@gmail.com`

Try these instead

Reply:
Thanks! That was exactly what I was wondering/missing